tests/integration-tests/tests/ultraserver/test_gb200/test_gb200/check_imex_status.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+
+# Maximum number of attempts
+MAX_ATTEMPTS=10
+
+# Function to run the command and check status
+verify_imex_is_up() {
+    local command="/usr/bin/nvidia-imex-ctl -N -j"
+    local file_prefix="result"
+
+    for ((i=1; i<=MAX_ATTEMPTS; i++)); do
+        echo "Attempt $i for IMEX"
+
+        command_output=$(${command} 2>/dev/null)
+        # Check if the status is UP
+        if echo "$command_output" | grep -q " \"status\": \"UP\""; then
+            echo "IMEX is UP. Exiting loop."
+            echo "writing the output in ${file_prefix}_${SLURM_JOB_ID}_$(hostname).out"
+            echo "$command_output" > ${file_prefix}_${SLURM_JOB_ID}_$(hostname).out 2> ${file_prefix}_${SLURM_JOB_ID}_$(hostname).err
+            break
+        elif [ $i -eq $MAX_ATTEMPTS ]; then
+            echo "Max attempts reached. Status is not UP."
```

Comment:
It may be useful to store the overall IMEX status in a file to facilitate troubleshooting in case of issues.

Reply:
I can print them in slurm-job_id files. But I dont see the point of keeping them in other files